src/components/VersionRestartModal/TaskStatusCheckbox.tsx

```diff
@@ -28,11 +27,9 @@ const CheckboxComponent: React.FC<TaskStatusCheckboxProps> = ({
     name={taskId}
     label={
       <StateItemWrapper>
-        <PaddedSquare color={mapTaskStatusToColor[status]} />
-        {baseStatus && (
-          <PaddedSquare color={mapTaskStatusToColor[baseStatus]} />
-        )}
-        {displayName}
+        <StyledTaskStatusIcon status={status} />
```

Comment:
We could give these icons a fixed width so if baseStatus doesn't exist the displayName text doesn't get shifted to the side.